mimesis/providers/date.py

```diff
@@ -176,10 +177,11 @@ def gmt_offset(self) -> str:
         """
         return self.random.choice(GMT_OFFSETS)
 
-    def datetime(self, humanized: bool = False, **kwargs) -> DateTime:
+    def datetime(self, humanized: bool = False, timezone: str = '', **kwargs) -> DateTime:
         """Generate random datetime.
 
         :param humanized: Readable representation.
+        :param timezone: Custom timezone.
```

Comment:
We need to add a note about `pytz` here. We should also mention that installing `pytz` is required to use it.